src/components/Card.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Card = props => {
+   const {title, genders, year, poster} = props;
+
+   
+
+  return (
+    <div className='card'>
+      <img src={poster} alt={title} className="card_poster"/>
+      <div className='card_top'>
+         <div className='card_title'>{title}</div>
+         <div className='card_year'>{year}</div>
+      </div>
+      <div className='card_genders'>{genders.toString()}</div>
```

Comment:
`genders.toString()` looks a bit odd and its result has no spaces in between the words. Consider using `Array.join` to build the result string.
Minor: typo genders -> genres.